src/os/bluestore/BlueStore.cc

```diff
@@ -4864,158 +4864,210 @@ void BlueStore::_init_logger()
-    "Average finishing state latency");
-  b.add_time_avg(l_bluestore_state_done_lat, "state_done_lat",
-    "Average done state latency");
-  b.add_time_avg(l_bluestore_throttle_lat, "throttle_lat",
-		 "Average submit throttle latency",
-		 "th_l", PerfCountersBuilder::PRIO_CRITICAL);
-  b.add_time_avg(l_bluestore_submit_lat, "submit_lat",
-		 "Average submit latency",
-		 "s_l", PerfCountersBuilder::PRIO_CRITICAL);
+
   b.add_time_avg(l_bluestore_commit_lat, "commit_lat",
-		 "Average commit latency",
-		 "c_l", PerfCountersBuilder::PRIO_CRITICAL);
-  b.add_time_avg(l_bluestore_read_lat, "read_lat",
-		 "Average read latency",
-		 "r_l", PerfCountersBuilder::PRIO_CRITICAL);
+    "Average commit latency",
+    "C_l", PerfCountersBuilder::PRIO_CRITICAL);
+  //****************************************
+
+  // Update Transaction stats
+  //****************************************
+  b.add_time_avg(l_bluestore_throttle_lat, "txc_throttle_lat",
+    "Average submit throttle latency",
+    "Th_l", PerfCountersBuilder::PRIO_CRITICAL);
+  b.add_time_avg(l_bluestore_submit_lat, "txc_submit_lat",
+    "Average submit latency",
+    "S_l", PerfCountersBuilder::PRIO_CRITICAL);
+  b.add_u64_counter(l_bluestore_txc, "txc_count", "Transactions committed");
+  //****************************************
+
+  // Read op stats
+  //****************************************
   b.add_time_avg(l_bluestore_read_onode_meta_lat, "read_onode_meta_lat",
     "Average read onode metadata latency");
   b.add_time_avg(l_bluestore_read_wait_aio_lat, "read_wait_aio_lat",
     "Average read latency");
-  b.add_time_avg(l_bluestore_compress_lat, "compress_lat",
-    "Average compress latency");
-  b.add_time_avg(l_bluestore_decompress_lat, "decompress_lat",
-    "Average decompress latency");
-  b.add_time_avg(l_bluestore_csum_lat, "csum_lat",
-    "Average checksum latency");
-  b.add_u64_counter(l_bluestore_compress_success_count, "compress_success_count",
-    "Sum for beneficial compress ops");
-  b.add_u64_counter(l_bluestore_compress_rejected_count, "compress_rejected_count",
-    "Sum for compress ops rejected due to low net gain of space");
+  b.add_time_avg(l_bluestore_csum_lat, "verify_csum_lat",
+    "Average checksum verification latency");
+  b.add_u64_counter(l_bluestore_read_eio, "read_eio",
+    "Read EIO errors propagated to high level callers");
+  b.add_u64_counter(l_bluestore_reads_with_retries, "reads_with_retries",
+    "Read operations that required at least one retry due to failed checksum validation");
+  b.add_time_avg(l_bluestore_read_lat, "read_lat",
+		 "Average read latency",
+		 "R_l", PerfCountersBuilder::PRIO_CRITICAL);
+  //****************************************
+
+  // kv_thread latencies
+  //****************************************
+  b.add_time_avg(l_bluestore_kv_flush_lat, "kv_flush_lat",
+    "Average kv_thread flush latency",
+    "Fl_l", PerfCountersBuilder::PRIO_INTERESTING);
```

Comment:
Any way to incorporate `K` into this counter to match the others? Possibly `Kfll`?

Reply:
may be kfsl?

Reply:
Sure